packages/minicaml/minicaml.transition/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/minicaml"
+license: "MIT"
+synopsis: "Project has been renamed to gobba. This virtual package can be safely removed"
+description: "Project has been renamed to gobba. This virtual package can be safely removed"
+version: "transition"
+
+depends: [
+    "gobba"
+]
+url {
+  src:
+    "https://github.com/0x0f0f0f/minicaml/releases/download/transition/minicaml-transition.tbz"
+  checksum: [
+    "sha256=9416e3ad0a93fe6882d98e3c3eb378db7801cc5b13f7c465fd79f951f8516e3a"
+    "sha512=b1fa01fe7378dcb230a28f0f824c0295a6172d7f75afaff5fc0f09520eb2f25bdf397b9463ea5bdbf8412dc6b40437b4b7276620f7bd0605c3ad16ad96717ea1"
+  ]
+}
```

Comment:
I'm not sure what's the goal of this transition release if older versions of minicaml (can you put back the tags btw? It's breaking all the old archives) don't exist anymore and the project has totally been renamed.

`url` fields are usually not necessary for this type of meta-packages anyway (see `jbuilder.transition` or `ocamldap.transition` as examples)